src/config.rs

```diff
@@ -174,6 +178,14 @@ impl Config {
         })
     }
 
+    #[must_use]
+    pub fn ui_url(&self) -> String {
+        // TODO: default to either vault.bitwarden.com or vault.bitwarden.eu based on the base_url?
```

Comment:
yeah, i think ideally we would follow the same pattern here that is used in `identity_url` and `notifications_url`